src/stackmap_checker/Makefile

```diff
@@ -25,6 +26,7 @@ bytecode: $(PREFIX)%.c
 $(TARGET_OBJS): $$(basename $$@).ll
 	$(LLC) -filetype=obj $<
 	$(LLC) -filetype=obj $<
+	rm /tmp/__stack_resizer_*
```

Comment:
Is this hard-coded filename intentional? Shouldn't it be relative to either a) the current dir b) a temp dir created by `mktemp`?

Reply:
It's intentional, I suppose. These files are created [here](https://github.com/gabi-250/llvm/blob/17433a9d340ef97bf02bae127abcd7ca3499ede3/lib/Target/X86/Utils/X86StackSizeComparator.cpp#L19). It's the ugly hack that ensures the stack size of a function is always the same as the stack size of its unoptimized copy. Unfortunately, to maintain state across runs, there needs to be a separate file for each function. The files need to be cleaned up in immediately after they are used.

Reply:
Putting "permanent"(ish) files in `/tmp/` is a security and reliability problem. Can we not make this file relative to the repository itself? Maybe as a `.something` file?

Reply:
That does sound like a better idea. I think the trouble was that the code that generates the files is nested somewhere in the `llvm` submodule and I didn't want to add a `../../../` relative path in there.